README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
This change doesn't seem to have a clear purpose. Is there a specific reason for adding ";wjkjewhkjeh" to the README?